master-firmware/src/scara/lie_groups.h

```diff
@@ -0,0 +1,31 @@
+#ifndef LIE_GROUPS_H
+#define LIE_GROUPS_H
+
+#include <aversive/math/vect2/vect2.h>
+#include <aversive/math/geometry/vect_base.h>
+
+// Special Orientation group 2 (SO2) defines a 2D rotation
+typedef struct {
+    float angle;
+} so2_t;
+
+so2_t so2(float angle);
+
+point_t so2_rotate(so2_t rotation, point_t point);
+point_t so2_inverse_rotate(so2_t rotation, point_t point);
+
+// Translation
+vect_t translation_2d(float x, float y);
+
+// Special Euclidian group 2 (SE2) defines a 2D rigid body transform (rotation + translation)
+typedef struct {
+    so2_t rotation;
+    vect_t translation;
+} se2_t;
+
+se2_t se2(float angle, vect_t translation);
```

Comment:
Can we rename this into something like `se2_create` or `se2_make` ? It would make the code easier to search.

Reply:
Done